### PR TITLE
LPS-134899 Fix last modified sort order on Process Builder on CE

### PR DESCRIPTION
**Pull request by MariMalta**

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
Hey @MariMalta,  there are some broken tests related to your changes. Could you please check this with the QA guys? Thx

File: modules/apps/portal-workflow/portal-workflow-kaleo-api/src/main/java/com/liferay/portal/workflow/kaleo/util/comparator/KaleoDefinitionVersionModifiedDateComparator.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public int compare(
 		Date modifiedDate1 = kaleoDefinitionVersion1.getModifiedDate();
 		Date modifiedDate2 = kaleoDefinitionVersion2.getModifiedDate();
 
-		int value = DateUtil.compareTo(modifiedDate1, modifiedDate2);
+		int value = DateUtil.compareTo(modifiedDate2, modifiedDate1);
```
Comment (rafaprax) [inline modules/apps/portal-workflow/portal-workflow-kaleo-api/src/main/java/com/liferay/portal/workflow/kaleo/util/comparator/KaleoDefinitionVersionModifiedDateComparator.java:51]: Hey @MariMalta, I don't know why we need to do this change here. I took a look at some `*ModifiedDateComparator` and everyone seems to use the same statement `DateUtil.compareTo(modifiedDate1, modifiedDate2)`. Probably the root cause may be in the above layers(DisplayContext and etc). Please check this so we can discuss more about the fix on slack.

Thanks Mari

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
This behaviour is the expected, the field's mask just confused us.

@MariMalta and @rafaprax, I'll close this pr.

thanks!

**Pull request closed without merging**